src/metabase/lib/drill_thru/common.cljc

```diff
@@ -1,8 +1,13 @@
 (ns metabase.lib.drill-thru.common
   (:require
+   [metabase.lib.card :as lib.card]
```

Comment:
1. New -> Question -> Orders -> Save as Q1
2. New -> Question -> Q1 -> Visualize
3. Click on User ID cell in first row > View details

An incorrect GET `/api/action?model-id=` request is triggered and fails (`model-id: "nullable value must be an integer greater than zero."`)

The request is not triggered when data source is a table.

Reply:
@kamilmielnik could you record a loom with repro steps? I can't repro a broken call.

Reply:
Fixed